GitUpdateScript.sh

```diff
@@ -94,26 +94,30 @@ rm Temp_dir_log #rm removes/dels the file
 }
 
 home_dir=$(pwd)  #v1 holds the home dir
-echo "Git Repositories Found:"
+
 
 # todo Count from find instead of iterating loop from file
-find -name .git| tee Temp_dir_log  #finds file with name .gitignore and stores results in file called FILENAME
 
-echo "Going to each git repositories and update:"
-while read file_dir
-do
 
-	GoToGitRepositoryDirectory
+arrayFileDir=($(find -name .git))
+numberOfGitRepos=$(find -name .git|wc -l)  #wc is word count -l makes it count lines, | is pipe and find finds all .git
+
+
+echo "Going to each git repository and updating:"
+
+for (( n=0; n<$numberOfGitRepos; n++ ))
+do
+    file_dir=${arrayFileDir[$n]}
+ 	GoToGitRepositoryDirectory
 
 	BranchChecking  #call branch checking function
 
 	git pull
 
 	echo "Elapsed Time:" $SECONDS
-done < Temp_dir_log
 
+done
 
-RemoveFileCreatedToHoldGitReposNames
```

Comment:
Add a script start comment at the bottom of the script

e.g. 
########
# Script start
########